docs-src/guide-rpc-server-api.scrbl

```diff
@@ -0,0 +1,333 @@
+#lang scribble/manual
+@(require "lib.rkt")
+
+@title[
+  #:version reach-vers
+  #:tag     "guide-rpc-server-api"
+  ]{The Reach RPC server API and how to use it}
+
+The Reach RPC server's design is deliberately lean and simple. In fact, it
+mostly just re-exports everything offered by the standard library via HTTPS
+under a corresponding @urlin{/stdlib/<function name>} route. Arguments and
+their order remain unchanged.
+
+@;TODO better example(?)
+For example, rather than leveraging the standard library directly like so:
+@js{ const balance = stdlib.formatCurrency('100000000000000000000', 18); }
+
+...an equivalent RPC request is issued, where the payload is a JSON-encoded
+array which represents ordered arguments to the target function:
+@js{ POST /stdlib/formatCurrency ['100000000000000000000', 18] }
+
+Likewise, Reach RPC @italic{@tech{frontends} and client libraries} are
+correspondingly lean and simple. Unless you're constructing a new client
+library (@seclink["guide-rpc-connect"]{more on this later}) for a language
+which doesn't yet have one available, you won't issue these RPC requests
+directly; you'll instead send such requests via a thin wrapper which has
+already been provided for you.
+
+@py{
+  # In this example we're using the Python RPC client library
+  balance = rpc('/stdlib/formatCurrency', '100000000000000000000', 18)
+}
+
+@italic{The remainder of this section introduces concepts discretely and with
+short code snippets. If you'd prefer to see fully-realized examples of
+@tech{RPC} @tech{frontends} in their entirety, it's recommended to browse the
+@urlin{client-<language>} directories @reachexlink["tut-6-rpc" "here"].}
+
+
+@;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+@section[#:tag "connecting"]{Connecting to the RPC server}
+
+By convention, @tech{RPC} @tech{frontends} issue all requests and receive all
+responses through only two functions.
+
+The simpler of the two is named @pyin{rpc}. @pyin{rpc}'s first argument is the
+route which represents a desired operation; the rest are forwarded to this
+operation as @italic{its} arguments.
+
+@py{
+  starting_balance = rpc('/stdlib/parseCurrency', 10)
+  acc_alice        = rpc('/stdlib/newTestAccount', starting_balance)
+  acc_bob          = rpc('/stdlib/newTestAccount', starting_balance)
+}
+
+The second - and more involved - function will be called something like
```

Comment:
This is the difficulty I'm talking about... you have to go between what the API is and then how it is "commonly" implemented. I think it is too heavy a load